src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java

```diff
@@ -226,12 +248,8 @@ public FormValidation doCheckApiURL(@QueryParameter String value)
 
         public FormValidation doCheckNamespace(@QueryParameter String value)
                 throws IOException, ServletException {
-        	// with some of the paths into the Image Stream SCM, the env vars typically available for the build steps are not available,
-        	// we we have to force the user to specify the project/namespace
-        	FormValidation fv = ParamVerify.doCheckNamespace(value);
-        	if (fv.kind == FormValidation.Kind.OK)
-        		return fv;
-        	return FormValidation.error("Please specify the name of the project");
+			// If a namespace is not specified, the default one is going to be used
+        	return ParamVerify.doCheckNamespace(value);
```

Comment:
Yeah, we were just not calling getNamespace(overrides) from the ImageStreams SCM. We are now.